apps/hubble-stats/app/layout.tsx

```diff
@@ -20,18 +24,72 @@ export default function RootLayout({
     setIsDarkMode('light');
   }, [setIsDarkMode]);
 
+  const items = [
+    {
+      name: 'Hubble',
+      isInternal: true,
+      href: '',
+      Icon: ContrastTwoLine,
+      subItems: [
+        {
+          name: 'Bridge',
+          isInternal: false,
+          href: 'https://app.webb.tools/#/bridge',
+        },
+        {
+          name: 'Explorer',
+          isInternal: true,
+          href: '',
+        },
+        {
+          name: 'Faucet',
+          isInternal: false,
+          href: 'https://develop--webb-faucet.netlify.app/',
+        },
+      ],
+    },
+    {
+      name: 'Tangle Network',
+      isInternal: true,
+      href: '',
+      Icon: Tangle,
+      subItems: [
+        {
+          name: 'DKG Explorer',
+          isInternal: false,
+          href: 'https://stats.webb.tools/#/keys',
+        },
+        {
+          name: 'Homepage',
+          isInternal: false,
+          href: 'https://tangle.webb.tools/',
+        },
+      ],
+    },
+  ];
+
+  const footer = {
+    name: 'Webb Docs',
+    isInternal: false,
+    href: 'https://docs.webb.tools/',
```

Comment:
Try to use the link inside webb-ui-kit, I think we have this link the the webb-ui-kit.